components/page-templates/MyAccountTemplate/MyAccountTemplate.tsx

```diff
@@ -142,21 +145,36 @@ const MyAccountTemplate = () => {
 
   const accordionData = [
```

Comment:
Why are we changing existing MyAccountTemplate ?

Reply:
@kibo-chandradeeptalaha No need to create a new template.
@kibo-sushant informed after discussion with @kibo-kevinwatts